package.json

```diff
@@ -12,11 +12,9 @@
     "@typescript-eslint/eslint-plugin": "latest",
     "@typescript-eslint/parser": "latest",
     "eslint": "^8.41.0",
-    "eslint-plugin-react": "latest"
-  },
-  "eslintConfig": {
-    "extends": "@redwoodjs/eslint-config",
```

Comment:
I think we should try to extend from the `@redwoodjs/eslint-config` package, even in our own `.eslintrc.json` file if possible.